librz/debug/p/debug_native.c

```diff
@@ -615,6 +615,14 @@ static RzList /*<RzDebugPid *>*/ *rz_debug_native_threads(RzDebug *dbg, int pid)
 #endif
 }
 
+RZ_API ut64 rz_debug_get_tls(RZ_NONNULL RzDebug *dbg, int tid) {
```

Comment:
Usually, if there is `RZ_NONNULL` for some argument, it should be accompanied by the `rz_retrun_if_fail()` or `rz_return_val_if_fail()` assert.